README.md

```diff
@@ -1 +1,251 @@
+        can_be_archived?() in the Item class
+            should return true if published_date is older than 10 years.
+            otherwise, it should return false.
+        move_to_archive() in the Item class
+            should reuse can_be_archived?() method.
+            should change the archived property to true if the result of the can_be_archived?() method is true.
+            should do nothing if the result of the can_be_archived?() method is false.
+        can_be_archived?() in the Book class
+            should override the method from the parent class.
+            should return true if parent's method returns true OR if cover_state equals to "bad".
+            otherwise, it should return false.
+        can_be_archived?() in the MusicAlbum class
+            should override the method from the parent class.
+            should return true if parent's method returns true AND if on_spotify equals true.
+            otherwise, it should return false.
+        can_be_archived?() in the Movie class
+            should override the method from the parent class.
+            should return true if parent's method returns true OR if silent equals true.
+            otherwise, it should return false.
+        can_be_archived?() in the Game class
+            should override the method from the parent class.
+            should return true if parent's method returns true AND if last_played_at is older than 2 years.
+            otherwise, it should return false.
+    Add unit tests for all implemented methods.
+
+User interface
+
+    Create a main.rb file that will serve as your console app entry-point.
+    Your console app, at the start, should:
+        Present the user with a list of options to perform.
+        Let users choose an option.
+        If needed, ask for parameters for the option.
+        Have a way to quit the app.
+    The following options should be available:
+        List all books
+        List all music albums
+        List all movies
+        List of games
+        List all genres (e.g 'Comedy', 'Thriller')
+        List all labels (e.g. 'Gift', 'New')
+        List all authors (e.g. 'Stephen King')
+        List all sources (e.g. 'From a friend', 'Online shop')
+        Add a book
+        Add a music album
+        Add a movie
+        Add a game
+    All data should be preserved by saving collections in .json files.
+
+Database
+
+    Create a schema.sql file with tables that will be analogical to the structure of the classes in your app. As you cannot implement inheritance in the database tables - add all properties and associations from the parent Item class as table columns to all tables based on the child classes. Thanks to that it can be used to store data in the future.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Project Presentation <a name="live-demo"></a>
```

Comment:
- [ ] the video presentation should be of 5mins maximum length, you can trim it to fit